### PR TITLE
SCJ-108: Add back deleted question

### PR DESCRIPTION
**Pull request by molund**
Added back a question that was accidentally deleted when SCJ-108 was implemented

When "Appeal" is selected:
- The hearing type dropdown should appear for criminal cases (e.g. 42024) <-- **this question was deleted**
- The "factum" question should appear for civil cases (e.g. 39000)

### CODE REVIEW

File: app/wwwroot/js/coa.js [hunk 1]
```diff
@@ -111,11 +114,12 @@ $(document).ready(function () {
     };
 
     $('#Appeal_FactumFiled input[type="radio"]').click(toggleAppeal);
+    $('#Appeal_FactumFiled input[type="radio"]').click(toggleAppeal);
```
Comment (duncan-oxd) [inline app/wwwroot/js/coa.js:117]: Is this meant to call `toggleAppeal` twice?

**Review by duncan-oxd — COMMENTED** [review]
Looks good to me; just one question about one of the lines in the js file

File: app/wwwroot/js/coa.js (hunk 1 above)
Comment (molund) [reply app/wwwroot/js/coa.js]: Good catch. I did not mean to call that twice.  I meant to call toggleAppeal when the dropdown value was changed but it seems to work okay without it, so I'll just delete the line.

**Review by duncan-oxd — APPROVED** [review]
looks good to me!

**Pull request merged**